client/CServerHandler.cpp

```diff
@@ -151,6 +157,7 @@ void CServerHandler::startLocalServerAndConnect()
 #ifdef VCMI_ANDROID
 	CAndroidVMHelper envHelper;
 	envHelper.callStaticVoidMethod(CAndroidVMHelper::NATIVE_METHODS_DEFAULT_CLASS, "startServer", true);
+	envHelper.Detach();
```

Comment:
In c# we have a conception of IDisposable. It looks like
using(something) {
Do something;
} // here all resources used by something are released.
This is quite clear and understandable. It will never get lost during refactoring. I wanted something of that kind. Some explicit thing telling that object should be destroyed ASAP.

Reply:
same here but just 
```
{
 	CAndroidVMHelper envHelper;
 	envHelper.callStaticVoidMethod(CAndroidVMHelper::NATIVE_METHODS_DEFAULT_CLASS, "startServer", true);
}
```
envHelper will be destructed when scope closes.

Reply:
Yes I know. I just feel it is not explicit enogh.